src/powershell-win-core/powershell-win-core.csproj

```diff
@@ -64,4 +57,6 @@
     <ProjectReference Include="..\Microsoft.Management.UI.Internal\Microsoft.PowerShell.GraphicalHost.csproj" Condition="'$(SDKToUse)' == 'Microsoft.NET.Sdk.WindowsDesktop' "/>
   </ItemGroup>
 
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
```

Comment:
Shall we move this above next to `<Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />`?

Reply:
We can try, but this was the recommendation from the .NET team member.